firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -3700,6 +3700,130 @@ extension String {
         comment: "Question shown to user when tapping an SMS or MailTo link that opens the external app for those.")
 }
 
+// MARK: Enhanced Tracking Protection/Unified Trust Panel
+extension String {
+    public static let EnhancedTrackingProtectionOnTitle = MZLocalizedString(
+        key: "Menu.EnhancedTrackingProtection.On.Title",
+        tableName: nil,
+        value: "Firefox is on guard",
```

Comment:
Instead of `Firefox`, we probably want to use `%@` and specify that the placeholder will be the app name.